docs/multisite/multisite_configuration.md

```diff
@@ -135,6 +135,20 @@ ibexa:
                             Identifier\ContentType: [article]
 ```
 
+Be aware of the fact that configurations in the different scopes are merged, not overwritten. This is important to have
+in mind when dealing with arrays:
```

Comment:
```suggestion
When working with arrays, configurations from different scopes are merged together. See the following example:
```

Reply:
Just a small remark: "See the following (...)" - new line.